### PR TITLE
Update model.py

### PR DESCRIPTION
**Pull request by PoeticDeath**
This fixes an incompatibility with python 3.9 on Windows, Possibly Linux as well.

### CODE REVIEW

**Comment by mattlyon93** [comment]
Not sure why this would be incompatible with python 3.9 unless you are changing the Keras / TensorFlow versions?

**Comment by PoeticDeath** [comment]
Using the default versions of all the requirements, Keras doesn't seem to find batch normalization when importing, switching it to the Keras within TensorFlow seems to fix this. I also only experienced this with python 3.9.
